src/transaction/signOperation.js

```diff
@@ -59,11 +65,17 @@ export const fromSignOperationEventRaw = (
 export const toSignOperationEventRaw = (
   e: SignOperationEvent
 ): SignOperationEventRaw => {
+  let signedOperation;
   switch (e.type) {
     case "signed":
+      signedOperation = e.signedOperation;
+      log(
+        "transaction-summary",
+        `✔️ has been signed! ${JSON.stringify(e.signedOperation)}`
+      );
```

Comment:
not a good idea to have log in a simple serialise function, it shouldn't do a side effect because we may use it for N usecases, plus on mobile we actually don't even use this i think. (as we don't need serialization)